bowling/tests/bowling_tests_gt.cpp

```diff
@@ -148,53 +148,102 @@ TEST_F(GameTests, getGameStatusShouldReturnFinishedWhenFramesAreFull) {
     EXPECT_EQ(game.getGameStatus(), Game::Status::FINISHED);
 }
 
+TEST_F(GameTests, getGameStatusShouldReturnFinishedWhenExtraFrames) {
+    file << "Robcio:12|27|X|1/|22|11|0-|0-|12|X||17\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::FINISHED);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnInProgressWhenAtLeastOnePlayerHasLessThan10Frames) {
+    file << "Robcio:" << tenStrikes << "|35\n";
+    file << "Grzegorz:" << tenStrikes << "\n";
+    file << "Eryk:12|27|X|0-|12|X|\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::IN_PROGRESS);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnFinishedWhenOnePlayerHas10FramesAndSecondHasExtraBalls) {
+    file << "Robcio:" << tenStrikes << "|35\n";
+    file << "Grzegorz:" << tenStrikes << "\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::FINISHED);
+}
+
+TEST_F(GameTests, getGameStatusShouldReturnInProgressWhenOnePlayerHasLessThan10FramesAndSecondHasExtraBalls) {
+    file << "Robcio:" << tenStrikes << "|35\n";
+    file << "Grzegorz:51|21|X\n";
+    file.flush();
+    game.loadFromFile(filePath);
+    EXPECT_EQ(game.getGameStatus(), Game::Status::IN_PROGRESS);
+}
+
 TEST_F(GameTests, countPointsFromVectorOfFramesWithoutStrikeNorSpareIncompleteGame) {
     std::vector<Frame> playerRolls{{'1', '2'}, {'4', '5'}, {'2', '2'}, {'7', '0'}};
-    EXPECT_EQ(game.countPoints(playerRolls), 23);
+    PlayerData player(defaultPlayerName, playerRolls);
+    EXPECT_EQ(player.countPoints(), 23);
 }
```

Comment:
this and 3 below tests can be changed to parametric. It is the same scenario.
Can you also prepare a test for counting Strikes only game? I have a gut feeling that your counting does not work properly.

Reply:
Your gut is right, we have a bug here..